### PR TITLE
add page home design

### PR DESCRIPTION
**Pull request by hsnice16**
### This PR will add code for the design of the Home page

In this PR, **227 additions 62 deletions** lines of code
- some files are only for assets
- some files contain only 5 to 6 lines of change

### Please review this PR and give your feedback

---

#### Also the design of all the pages is completed. It would be really nice if you can suggest any improvement on whole app.

image
image

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Good small PR and code clean coding. Good to merge

File: src/pages/bookmarks/Bookmarks.jsx [hunk 1]
```diff
@@ -50,16 +49,16 @@ export const Bookmarks = () => {
...
       </PageHeading>
 
-      <EmptyBookmark
+      {/* this commented code will get removed in subsequent PR's */}
+
+      {/* <EmptyBookmark
         imgSrc={emptyBookmark}
         imgAlt="empty bookmark"
         h1Text="Want to read a Broadcast later?"
         h2Text="Bookmark that"
```
Comment (b0llu) [inline src/pages/bookmarks/Bookmarks.jsx:58]: Remove commented code when done

File: src/pages/profile/Profile.jsx [hunk 2]
```diff
@@ -68,21 +70,21 @@ export const Profile = () => {
...
-      </Button> */}
+      </Button>
 
-      <Button
+      {/* <Button
         variant="outlined"
         onClick={() => setOpenEditProfileDialog(true)}
         className={classNames(styles.btn, styles.btn_unfollow)}
       >
         Unfollow
```
Comment (b0llu) [inline src/pages/profile/Profile.jsx:86]: Remove commented code when done

File: src/pages/profile/Profile.jsx [hunk 3]
```diff
@@ -137,16 +139,16 @@ export const Profile = () => {
...
       </Box>
 
-      <EmptyBookmark
+      {/* this commented code will get removed in subsequent PR's */}
+
+      {/* <EmptyBookmark
         imgSrc={noBroadcasts}
         imgAlt="no broadcasts"
         h1Text="Something's going in your mind?"
         h2Text="Broadcast it"
```
Comment (b0llu) [inline src/pages/profile/Profile.jsx:148]: Remove commented code when done

**Review by b0llu — APPROVED** [review]
Just remove commented code when done. Really liked that you used exports for svg. Great code you can merge this!

**Comment by hsnice16** [comment]
Thank you for reviewing,

**Pull request merged**